experiment-trait-data-visualizer/cache-refresh.R

```diff
@@ -0,0 +1,126 @@
+library(dplyr)
+library(tools)
+library(lubridate)
+options(scipen=999)
+
+# set up remote connection to BETYdb
```

Comment:
@nickheyek it wasn't clear from the README how to set these up. is it just putting 

```sh
export bety_user=bety
export bety_password=...
...
```

in .bashrc? 

Since access is restricted by ssh, I just hard coded the values. (see below). Does that make sense?